### ISSUE TITLE
SCOPING: Allow making studies hidden unless eligible

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#651]
# TL;DR
Make it possible for a researcher to specify that a study should be advertised to eligible participants, and shown to logged-in families on the studies page, but NOT displayed on the studies page to people who are not logged in or who do not have an eligible child.

# Narrative
As a researcher, I may sometimes have a study that depends on participation in a previous study and not want it to be displayed publicly - but I do want to automatically advertise it to participants who are eligible!

<details>
<summary>Feature Request fields</summary>
<p>
**With this `details` wrapper, you can just use this ticket to record your findings.
When you're ready to turn this into an implementation request, delete the scoping label and
this wrapper, leaving only the filled out information.**

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes
Leave important notes/suggestions here for implementation (important files, suggested approaches, etc.)

</p>
</details>

**Comment by mekline** [comment]
(Handled with blacklist/whitelist study criteria field)
